VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -135,21 +131,16 @@ public void onActivityCreated(Bundle savedInstanceState) {
             if ( courseData == null )
                 return;
 
-            String headerImageUrl = courseData.getCourse().getCourse_image(environment.getConfig());
-            NetworkImageView headerImageView = (NetworkImageView)getView().findViewById(R.id.header_image_view);
-            DisplayMetrics displayMetrics = getResources().getDisplayMetrics();
-            float screenWidth = displayMetrics.widthPixels;
-            float ideaHeight = screenWidth * 9 / 16;
-
-            headerImageView.setLayoutParams(new LinearLayout.LayoutParams(
-                LinearLayout.LayoutParams.MATCH_PARENT, (int) ideaHeight));
-            headerImageView.requestLayout();
-
-            headerImageView.setImageUrl(headerImageUrl, environment.getImageCacheManager().getImageLoader() );
+            final String headerImageUrl = courseData.getCourse().getCourse_image(environment.getConfig());
+            Glide.with(CourseDashboardFragment.this)
+                    .load(headerImageUrl)
+                    .placeholder(R.drawable.edx_map_login)
+                    .transform(new TopAnchorFillWidthTransformation(getActivity()))
+                    .into(headerImageView);
 
             courseTextName.setText(courseData.getCourse().getName());
             CourseEntry course = courseData.getCourse();
-            courseTextDetails.setText( course.getDescription(this.getActivity()));
+            courseTextDetails.setText(course.getDescription(this.getActivity()));
```

Comment:
I think the `this.getActivity()` can be replaced with just `getActivity()` here; there is no ambiguity in the context to be resolved.